[2022-02-03 17:28:45,215] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_gcs_dag.download_dataset_task scheduled__2020-12-02T00:00:00+00:00 [queued]>
[2022-02-03 17:28:45,252] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_gcs_dag.download_dataset_task scheduled__2020-12-02T00:00:00+00:00 [queued]>
[2022-02-03 17:28:45,253] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 17:28:45,253] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-03 17:28:45,253] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 17:28:45,279] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2020-12-02 00:00:00+00:00
[2022-02-03 17:28:45,285] {standard_task_runner.py:52} INFO - Started process 3330 to run task
[2022-02-03 17:28:45,292] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yellow_taxi_gcs_dag', 'download_dataset_task', 'scheduled__2020-12-02T00:00:00+00:00', '--job-id', '134', '--raw', '--subdir', 'DAGS_FOLDER/yellow_taxi_gcs_dag.py', '--cfg-path', '/tmp/tmpbbws2_8t', '--error-file', '/tmp/tmp1avdo26t']
[2022-02-03 17:28:45,299] {standard_task_runner.py:77} INFO - Job 134: Subtask download_dataset_task
[2022-02-03 17:28:45,466] {logging_mixin.py:109} INFO - Running <TaskInstance: yellow_taxi_gcs_dag.download_dataset_task scheduled__2020-12-02T00:00:00+00:00 [running]> on host 7e458922b35d
[2022-02-03 17:28:45,571] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-03 17:28:45,672] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yellow_taxi_gcs_dag
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2020-12-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-12-02T00:00:00+00:00
[2022-02-03 17:28:45,673] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-03 17:28:45,674] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2020-12.csv > /opt/***/output_2020-12.csv']
[2022-02-03 17:28:45,735] {subprocess.py:85} INFO - Output:
[2022-02-03 17:28:58,770] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-03 17:28:58,830] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yellow_taxi_gcs_dag, task_id=download_dataset_task, execution_date=20201202T000000, start_date=20220203T172845, end_date=20220203T172858
[2022-02-03 17:28:58,882] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-03 17:28:59,038] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-20 00:31:43,575] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_gcs_dag.download_dataset_task scheduled__2020-12-02T00:00:00+00:00 [queued]>
[2022-02-20 00:31:43,588] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_gcs_dag.download_dataset_task scheduled__2020-12-02T00:00:00+00:00 [queued]>
[2022-02-20 00:31:43,588] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-20 00:31:43,588] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-20 00:31:43,588] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-20 00:31:43,601] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2020-12-02 00:00:00+00:00
[2022-02-20 00:31:43,607] {standard_task_runner.py:52} INFO - Started process 2497 to run task
[2022-02-20 00:31:43,611] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yellow_taxi_gcs_dag', 'download_dataset_task', 'scheduled__2020-12-02T00:00:00+00:00', '--job-id', '505', '--raw', '--subdir', 'DAGS_FOLDER/yellow_taxi_gcs_dag.py', '--cfg-path', '/tmp/tmpf_ntwyv9', '--error-file', '/tmp/tmp4mvwcfif']
[2022-02-20 00:31:43,612] {standard_task_runner.py:77} INFO - Job 505: Subtask download_dataset_task
[2022-02-20 00:31:43,675] {logging_mixin.py:109} INFO - Running <TaskInstance: yellow_taxi_gcs_dag.download_dataset_task scheduled__2020-12-02T00:00:00+00:00 [running]> on host da07aa254901
[2022-02-20 00:31:43,720] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-20 00:31:43,752] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yellow_taxi_gcs_dag
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2020-12-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-12-02T00:00:00+00:00
[2022-02-20 00:31:43,755] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-20 00:31:43,756] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2020-12.csv > /opt/***/output_2020-12.csv']
[2022-02-20 00:31:43,774] {subprocess.py:85} INFO - Output:
[2022-02-20 00:32:01,479] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-20 00:32:01,509] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yellow_taxi_gcs_dag, task_id=download_dataset_task, execution_date=20201202T000000, start_date=20220220T003143, end_date=20220220T003201
[2022-02-20 00:32:01,535] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-20 00:32:01,585] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
